app/views/includes/footer.nunjucks.old

```diff
@@ -0,0 +1,17 @@
+<div class="global-footer">
+  <div class="global-footer__inner">
+
```

Comment:
Can this blank line be deleted if it isn't required?

Reply:
File shouldn't be there, removed...